paramak/parametric_components/blanket_poloidal_segment.py

```diff
@@ -0,0 +1,316 @@
+import numpy as np
+from scipy.interpolate import interp1d
+from scipy.optimize import minimize
+import sympy as sp
+import warnings
+
+from paramak import BlanketFP, RotateStraightShape
+from paramak.utils import cut_solid, rotate, extend, \
+    distance_between_two_points
+
+
+class BlanketFPPoloidalSegments(BlanketFP):
+    """Poloidally segmented Blanket inheriting from paramak.BlanketFP.
+
+    Args:
+        segments_angles (list, optional): If not None, the segments ends will
+            be located at these angles. If None and if the constraints
+            length_limits and nb_segments_limits are not None, segments angles
+            will be linearly distributed. Else, an optimum configuration
+            meeting the set requirements will be found. Defaults to None.
+        num_segments (int, optional): Number of segments (igored if
+            segments_angles is not None). Defaults to 7.
+        length_limits ((float, float), optional): The minimum and maximum
```

Comment:
```suggestion
        length_limits (list of (float, float), optional): The minimum and maximum
```

Reply:
perhaps this is always a list of tuples

Reply:
This argument doesn't work like that. In fact the user sets a min and max value for ALL the segments.
Does these min/max values must be set for each segments ? That'll be tricky if the number of segment isn't known beforehand.

Reply:
![image](https://user-images.githubusercontent.com/40028739/98525502-d632b780-2278-11eb-896f-daf7547af237.png)
This would be the mathematical description of the optimiser results

Reply:
ah yes min and max for all segments is best. I understand now